src/playback/view/playbacktoolbarmodel.h

```diff
@@ -40,6 +40,7 @@ class PlaybackToolBarModel : public QAbstractListModel, public async::Asyncable
     INJECT(playback, IPlaybackController, playbackController)
     INJECT(playback, workspace::IWorkspaceManager, workspaceManager)
 
+    Q_PROPERTY(bool floating READ floating WRITE setFloating NOTIFY floatingChanged)
```

Comment:
I'm not sure "floating" is the best name for this property but at least you should rename it to "isFloating" to provide uniformity with "isPlayAllowed" :)

Reply:
I renamed it to "isToolbarFloating" :)